_pages/release-strategies.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Releasing Software
+sidenav: approach
+sticky_sidenav: true
+---
+
+Releasing software is hard. Over time, we have developed strategies that can help reduce risk and uncertainity in the release process.
+
+## Small, Iterative Releases
+
+We recommend on all projects to get into the habit of releasing software as early and often as possible. The first
+iteration of your software should not aim to have full functionality; a better goal for the very first release is to have as little
+functionality as possible.
+
+## Big Bang Releases
+
+We do not recommend "big bang releases," defined here as when a team works many months on some code with the expectation that it will be
+"turned on" in a big release event. 
```

Comment:
i want to create an issue that talks about this more since i think the why behind this is realllllly important. but i think a placeholder link is a great idea

Reply:
Prior stuff: https://www.google.com/search?q=18f+iterative+release&oq=18f+iterative+release